lib/active_merchant/billing/gateways/reach.rb

```diff
@@ -0,0 +1,176 @@
+require 'base64'
+
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class ReachGateway < Gateway
+      # TODO: Things to check
+      # * The list of three digit fractions but only accept 2
+      # * Not sure the list of countries and currencies
```

Comment:
Are these todos still true?

Reply:
yup @curiousepic, need to clarify that on grooming meeting